### PR TITLE
Always return a new ValueNotification on value notification events

### PR DESCRIPTION
**Pull request by raman325**
Addresses comment from #151

### CODE REVIEW

File: zwave_js_server/model/node.py [hunk 1]
```diff
@@ -454,12 +454,11 @@ def handle_value_removed(self, event: Event) -> None:
     def handle_value_notification(self, event: Event) -> None:
         """Process a node value notification event."""
         # append metadata if value metadata is available
+        value_notification = ValueNotification(self, event.data["args"])
         value = self.values.get(_get_value_id_from_dict(self, event.data["args"]))
         if value:
             value.update(event.data["args"])
```
Comment (raman325) [inline zwave_js_server/model/node.py:460]: So the only open question that I am not clear on is whether we still need this line

**Review by MartinHjelmare — APPROVED** [review]
Looks good!

File: zwave_js_server/model/node.py (hunk 1 above)
Comment (raman325) [reply zwave_js_server/model/node.py:460]: @marcelveldt sorry to keep tagging you, I just didn't fully understand your comment in the other PR. Should we update the `Value` or should we not? I think from a practical standpoint what we'd be updating is the stored value of the `Value`

Comment (raman325) [reply zwave_js_server/model/node.py]: given that this doesn't change functionality by leaving it in, I think we can merge and address this separately if needed

**Pull request merged**